From b77e43af30034d7c5d7f51e3f7e17b13f0af9b5e Mon Sep 17 00:00:00 2001
From: Jackie Huang <jackie.huang@windriver.com>
Date: Fri, 13 Jul 2012 13:14:01 +0800
Subject: [PATCH 1/1] lm_sensors-pci-not-required

If there is no PCI bus on the board, don't die directly,
print the error message and continue the autodetection.

reference:
https://bugzilla.redhat.com/show_bug.cgi?id=806364

---
 prog/detect/sensors-detect |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/prog/detect/sensors-detect b/prog/detect/sensors-detect
index a060e26..1f735e5 100755
--- a/prog/detect/sensors-detect
+++ b/prog/detect/sensors-detect
@@ -2996,7 +2996,7 @@ sub read_sys_dev_pci
 	my ($dev, @pci_list);
 
 	opendir(local *DEVICES, "$devices")
-		or die "$devices: $!";
+		or print("$devices: $!\n") and return \@pci_list;
 
 	while (defined($dev = readdir(DEVICES))) {
 		my %record;
-- 
1.7.4

